sfge/src/main/java/com/salesforce/graph/vertex/UserClassVertex.java

```diff
@@ -75,6 +75,13 @@ public boolean isVirtual() {
         return ((ModifierNodeVertex) getOnlyChild(ASTConstants.NodeType.MODIFIER_NODE)).isVirtual();
     }
 
+    /**
+     * TODO: Check whether this class can be safely made to extend {@link FieldWithModifierVertex}.
+     */
+    public boolean isGlobal() {
```

Comment:
In a perfect world, I'd have been able to actually do the check for this myself. But in the interest of time, I opted not to do so.
Maybe we should add a tech-debt item to the backlog representing this?

Reply:
How about adding a superclass layer to `FieldWithModifierVertex` with a more generic name that has `isGlobal()` method and extending `UserClassVertex` and `UserInterfaceVertex`?

Reply:
That might work. I'll give that a shot.